custom/ilsgateway/tanzania/handlers/soh.py

```diff
@@ -52,6 +52,7 @@ def _cleanup(s):
 class SohFormatter(Formatter):
 
     def format(self, text):
+        text = text.replace('\r', ' ').replace('\n', ' ').strip()
```

Comment:
should we apply this at a higher level in the custom ils handler so that it works for all ils keywords?